services/agent/src/simcore_service_agent/core/_routes.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from fastapi import APIRouter, Depends, HTTPException, status
+
+from ..modules.task_monitor import TaskMonitor
+from ._dependencies import get_task_monitor
+
+router = APIRouter()
+
+
+@router.get("/health")
+def health(task_monitor: TaskMonitor = Depends(get_task_monitor)) -> dict[str, Any]:
+    # TODO: application health should b
+    # e pulled from some metrics like the number of fails
+    # from the monitoring system, if more thank X tasks fail in the last X, it needs to be marked as unhealthy
+
+    if not task_monitor.is_running:
+        raise HTTPException(status.HTTP_503_SERVICE_UNAVAILABLE, detail="unhealthy")
+
+    return dict(task_monitor=task_monitor.is_running)
```

Comment:
This response is not very useful. Is like returning ``{ "task_monitor": True }`` always

Reply:
The healtcheck  is used to check whether is healthy or not. 
Therefore, just use the status_codes for that. i.e. status_code=200 is health otherwise unhealthy.
Sometimes we return some static info but today i would even advice against ... 
For info we can have a separate entrypoint (e.g. ``/meta`` or ``/info`` )